src/app/styles.ts

```diff
@@ -106,22 +108,26 @@ export const TextContainer = styled.div`
   z-index: 1;
   gap: clamp(10px, 5vh, 30px);
 `;
+
 export const ImageBackground = styled.div`
   position: absolute;
   top: 0;
   left: 0;
   height: 100%;
   width: 100%;
-  background-color: rgba(40, 120, 194, 0.5);
+  background: linear-gradient(#0461b6, #002b52);
+  opacity: 0.55;
   z-index: 0;
 `;
+
 export const ImageContainer = styled.div`
   position: absolute;
   width: 100%;
   height: 100svh;
   top: 0;
   left: 0;
 `;
```

Comment:
we'll probably have to adjust this once we add the navbar right?

Reply:
maybe, depending on how the navbar affects the design

Reply:
cc @kyrenetam